packages/react/src/file-upload/FileUpload.test.tsx

```diff
@@ -54,4 +54,15 @@ describe('FileUpload', () => {
 			expect(await axe(container)).toHaveNoViolations();
 		});
 	});
+
+	describe('Max files', () => {
+		it('throws expected error', () => {
```

Comment:
Jest has some builtin handlers for asserting thrown errors. e.g. `toThrowError`. Here you could do:

```ts
expect(() => render(<FileUploadExample maxFiles={0} />)).toThrowError(
		'maxFiles cannot be less than 1'
);
```

Do you think a `console.warn` would be safer in the component, as opposed to throwing the error?

Reply:
Removed this test since it now just throws a warning instead.